src/lib/y2storage/fake_device_factory.rb

```diff
@@ -694,12 +694,7 @@ def create_subvolume(parent, args)
       raise ArgumentError, "No path for subvolume" unless path
 
       blk_device = BlkDevice.find_by_name(@devicegraph, parent)
-      btrfs = blk_device.filesystem
-      parent_subvol = find_default_subvolume(btrfs) || btrfs.top_level_btrfs_subvolume
```

Comment:
So now the `find_default_subvolume` method is not used anymore. We should delete it.